torch/utils/data/datapipes/iter/routeddecoder.py

```diff
@@ -41,7 +40,6 @@ def __init__(self,
         if not handlers:
             handlers = (decoder_basichandlers, decoder_imagehandler('torch'))
         self.decoder = Decoder(*handlers, key_fn=key_fn)
-        deprecation_warning_torchdata(type(self).__name__)
```

Comment:
@ejguan Is the plan to keep this DataPipe here in Core rather than TorchData?

Reply:
We should keep deprecation warning here.